__tests__/app.test.js

```diff
@@ -88,6 +88,22 @@ describe("/api/articles", () => {
             expect(body.articles).toBeSortedBy("created_at")
         ))
     })
+    test("GET: 200 - take a sort_by query and respond with articles sorted by given column name", () => {
+        return request(app)
+        .get("/api/articles?sort_by=author")
+        .expect(200)
+        .then(({body})=>(
+            expect(body.articles).toBeSortedBy("author")
+        ))
+    })
```

Comment:
This ticket also includes testing for order query and should include another 200 test for order=desc or order=asc